.changeset/strong-colts-dance.md

```diff
@@ -0,0 +1,28 @@
+---
+'xstate': major
+---
+
+The `each()` action creator has been added for SCXML compatibility:
+
+```js
+// ...
+context: {
+  todos: [/* ... */],
+  todo: null, // will be populated by `each` action
+  currentTodo: null // will be populated by `each` action
+}
+actions: each([
+  (context, event) => {
```

Comment:
This looks a little bit different from other actions - it's actually very different (it's an aggregate action~) from most so this might very well be justified. I'm wondering what are your thoughts about either:
- flipping args - a short `{array, item}` would be easier to spot before potentially lengthy actions
- or going full object style - `each({ array, item, actions })`

Reply:
I'd probably lean towards the second suggestion (full object style) 🤔